ompi/runtime/ompi_rte.c

```diff
@@ -1176,27 +1181,36 @@ static int _setup_job_session_dir(char **sdir)
 {
     /* get the effective uid */
     uid_t uid = geteuid();
+    int rc;
```

Comment:
I didn't write this code, and I would rather not modify it beyond fixing the one thing that is required. If someone else wants to rewrite the entire thing, they are welcome to do so.

Reply:
6b4fb509e92 maybe ?

Reply:
No - that was my cleanup attempt. Somebody else wrote the code that defined the session directories when they aren't provided. Regardless, I don't know why they chose to set the target location to NULL instead of the param, but it effectively doesn't matter as the function is only used to set that target location. 🤷‍♂️ 

If you feel strongly about it, I'm happy to withdraw the PR and let someone else address it. Truly doesn't impact me at all.